python/src/main/resources/grpc/python/zeppelin_python.py

```diff
@@ -0,0 +1,107 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Only `PyZeppelinContext` is duplicated. If IPythonInterpreter become mature enough, I think we can remove the old implementation.